saleor/core/models.py

```diff
@@ -127,3 +127,20 @@ class Job(models.Model):
 
     class Meta:
         abstract = True
+
+
+class EventPayload(models.Model):
+    payload = JSONField(default=dict, encoder=CustomJsonEncoder)
+
+
+class EventPayloadReference(models.Model):
```

Comment:
This model represents a single Celery task that sends a webhook, so maybe a better name would be `EventTask`?